ui/src/containers/NodeVolumesUtils.test.js

```diff
@@ -0,0 +1,256 @@
+import { isVolumeDeletable } from './NodeVolumesUtils';
+
+const testcase1 = {
```

Comment:
We probably could have more meaningful name that `testcaseX`, `testcaseY`, …

- `testcase1` => `testcaseVolumeWithoutStatus`
- `testcase2` => `testcaseVolumePending`
- `testcase3` => `testcaseVolumeTerminating`
- `testcase4` => `testcaseVolumeFailedWithoutPv`
- `testcase5` => `testcaseVolumeFailedWithPvFailed`
- […]
- `testcase11` => `testcaseVolumeAvailableWithPvFailed`
- […]
- `testcase17` => testcaseVolumeAvailableWithoutPv